Libs/MRML/Core/vtkCacheManager.cxx

```diff
@@ -641,13 +641,9 @@ int vtkCacheManager::ClearCache()
   {
     this->MarkNodesBeforeDeletingDataFromCache ( this->RemoteCacheDirectory.c_str() );
     vtksys::SystemTools::RemoveADirectory ( this->RemoteCacheDirectory.c_str() );
-  }
-#if (VTK_MAJOR_VERSION >= 9 && VTK_MINOR_VERSION >= 0 && VTK_BUILD_VERSION >= 20210806)
+    }
   if ( !vtksys::SystemTools::MakeDirectory ( this->RemoteCacheDirectory.c_str() ) )
-#else
-  if ( vtksys::SystemTools::MakeDirectory ( this->RemoteCacheDirectory.c_str() ) == false )
-#endif
-  {
+    {
```

Comment:
```suggestion
  {
```

This will be applied in upcoming `remove-support-for-vtk-9.1-or-older` topic contributed as an independent pull request.